integration-tests/azure-eventhubs/README.adoc

```diff
@@ -0,0 +1,18 @@
+== Azure Azure Event Hubs Service integration tests
```

Comment:
Maybe an extra Azure above. The prerequisites section is really nice.

Reply:
Yes, thanks!